pkg/apis/resource/types.go

```diff
@@ -172,6 +172,10 @@ type ResourceHandle struct {
 	// plugin should be invoked to process this ResourceHandle's data once it
 	// lands on a node. This may differ from the DriverName set in
 	// ResourceClaimStatus this ResourceHandle is embedded in.
+	//
+	// The special "nop.k8s.io" constant may be used here. This tells the
+	// kubelet that it does no resource driver is required when starting or
+	// stopping containers that use this resource.
```

Comment:
The alternative that I had considered was something like:
```
Usage []string
```

where the empty set is the default behavior (call plugin). The same could also be specified explicitly with `usage: [ cdi ]`.

`usage: [ nop ]` could indicate "do nothing".

If we ever do something with CNI, `usage: [ cni ]` could be added, perhaps even in combination (`usage: [ cdi, cni ]`).

I got stuck a bit when wondering whether all strings need to be pre-defined. Probably, otherwise we have a problem when a future release adds a new value that an old kubelet doesn't support. We want that old kubelet to fail when encountering an unknown value instead of silently ignoring it.